web/packages/teleport/src/Discover/Server/EnrollEc2Instance/Ec2InstanceList.tsx

```diff
@@ -175,8 +175,15 @@ export const Ec2InstanceList = ({
                 <Text bold>Configure your AWS IAM permissions</Text>
                 <Text typography="subtitle1" mb={3}>
                   We were unable to list your EC2 instances. Run the command
-                  below on your AWS CloudShell to configure your IAM
-                  permissions. Then press the refresh button above.
+                  below on your{' '}
+                  <Link
+                    href="https://console.aws.amazon.com/cloudshell/home"
+                    target="_blank"
+                  >
+                    Amazon CloudShell
```

Comment:
Shouldn't we call it AWS CloudShell? I think this [the official product name](https://aws.amazon.com/cloudshell/).

Reply:
Yeah, good catch :+1: 
I've replace both occurrences of this term